actors/builtin/reward/reward_actor.go

```diff
@@ -115,6 +115,7 @@ func (a *RewardActor) AwardBlockReward(
 	rt vmr.Runtime,
 	miner addr.Address,
 	penalty abi.TokenAmount,
+	gasReward abi.TokenAmount,
 	minerNominalPower abi.StoragePower,
 	currPledge abi.TokenAmount,
```

Comment:
Please document these parameters. For gas reward, include the note that the gas reward value is expected to have been already transferred to this actor by the runtime priori to invoking this method.